src/neptune/metadata_containers/metadata_container.py

```diff
@@ -380,6 +380,20 @@ def stop(self, *, seconds: Optional[Union[float, int]] = None) -> None:
             self._state = ContainerState.STOPPED
             self._forking_cond.notify_all()
 
+    def get_state(self) -> str:
+        """Returns the current state of the container as a string.
```

Comment:
At least for now, _container_ is internal terminology

```suggestion
        """Returns the current state of the object as a string.
```